static/title-bar.less

```diff
@@ -0,0 +1,35 @@
+@import "./variables/ui-variables";
+
+.title-bar {
+  height: 23px;
+
+  flex-shrink: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  font-family: -apple-system, BlinkMacSystemFont, "Helvetica Neue";
```

Comment:
Maybe we don't have to force a `font-family` and just let it inherit whatever the theme defined. The One themes use the system font, but some themes might wanna use something else.